src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.DefaultItems.props

```diff
@@ -16,8 +16,8 @@ Copyright (c) .NET Foundation. All rights reserved.
              And '$(EnableDefaultNoneItems)' == 'true'
              And '$(AppDesignerFolder)' != ''">
 
-    <_WebToolingArtifacts Include="$(AppDesignerFolder)\launchSettings.json;
-                          $(AppDesignerFolder)\serviceDependencies.json;
+    <_WebToolingArtifacts Include="$(AppDesignerFolder)\launchSettings*.json;
```

Comment:
You don't need the wildcard as there is just launchsettings.json. The wildcard could pick up unintended files

Reply:
The problem with the actual path to a file is that if the file is absent, project system marks them as non-existent files. Wildcard does an exist check as well. I thought about the additional files getting included in none that starts with "launchSettings" and have ".json" extension but if such a file exist, it should ideally be a none item any ways. 

The other alternative is to add an exist check to all files without the wildcard but in this case since it is moving to None, thought a wildcard include on the file with the same name is more appropriate.

Reply:
![image](https://user-images.githubusercontent.com/3981619/87808401-7ec29480-c80e-11ea-9775-026a9d4e8730.png)

Reply:
This is because these get included in None, got it. Looks good to me